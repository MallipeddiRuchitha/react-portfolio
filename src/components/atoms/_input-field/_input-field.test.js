import React from 'react';
import { create } from "react-test-renderer";
import { render ,fireEvent} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import InputField from './_input-field';

describe("InputField component",() =>{
    test("Matches the snapshot",() => {
        const inputField = create(<InputField />);
        expect(inputField.toJSON()).toMatchSnapshot();

    })
    test("inputField element is rendered",() => {
        const { getByTestId, getByText } = render(<InputField />);
        expect(getByTestId("inputField")).toBeInTheDocument();
        
    });
   
   

});