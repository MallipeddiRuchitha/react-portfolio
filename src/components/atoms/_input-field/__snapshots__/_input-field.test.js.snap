// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputField component Matches the snapshot 1`] = `
<div
  className="makeStyles-styleInputFields-1"
  id="inputField"
>
  <WithStyles(ForwardRef(Input))
    inputProps={
      Object {
        "data-testid": undefined,
      }
    }
    required={true}
    style={
      Object {
        "color": "#000",
      }
    }
  />
</div>
`;
